controllers/secrets-webhook/secretswebhook_test.go

```diff
@@ -0,0 +1,134 @@
+package secrets_webhook
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	log "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func TestInjectSecretsc(t *testing.T) {
```

Comment:
Could we possibly take a look at the test names? I think this one might be better described by something like `Test_mutatePods_OneSecret` or similar. Tests normally have `Test_` as a prefix, and it is good for the test name to explicitly mention the function name.